client/packages/openblocks/src/comps/comps/tableComp/tableComp.tsx

```diff
@@ -420,6 +420,17 @@ TableTmpComp = withMethodExposing(TableTmpComp, [
       }
     },
   },
+  {
+    method:{
+      name: "clearSelections",
```

Comment:
The method reset the selection to the default value,  so it better rename to "resetSelections"

Reply:
Thanks, changed the method name